models/DataObject/ClassDefinition/Data/Table.php

```diff
@@ -643,4 +636,25 @@ public function isEqual($oldValue, $newValue): bool
     {
         return $this->isEqualArray($oldValue, $newValue);
     }
+
+    public function getParameterTypeDeclaration(): ?string
+    {
+        return '?array|string';
```

Comment:
I don't think this will work (before PHP8, and even there it would have to be `array|string|null`).

Note that `getPhpdocInputType()` doesn't return a nullable type.